version1/04 - Web services.md

```diff
@@ -1800,6 +1800,63 @@ Content-type: application/json
 - **servicePermissions**
   - the group service permissions
 
+### Get all objects explicit permissions for access groups by ID
+
+Retrieves a list of all objects with their respective explicit permissions for a certain access group
+
+**Request**
+
+```text
+GET /access-group/{id}/permissions?offset=0&limit=100
+Accept: application/json
+Authorization: Bearer {token}
+```
+
+###### Details
+
+- **id**
+  - the identifier of the access group to fetch
+- **offset (optional)**
+  - return objects from the specified offset
+- **limit (optional)**
+  - return n number of objects per page (maximum 100 and 10 by default)  
+
+**Response**
+
+200 OK
+Content-type: application/json
+{
+"results": [
+         {
+             "accessGroupId": int,
+             "explicitPermissions": [string],
+             "discriminator": string,
+             "objectId": int
+         },
+         {
+             "accessGroupId": int,
+             "explicitPermissions": [string],
+             "discriminator": string,
+             "listType": string
+         },
+         {
+             "accessGroupId": int,
+             "explicitPermissions": [string],
+             "discriminator": string,
+             "listType": string,
+             "code": string
+         }
+     ],
+     "hasMore": true
```

Comment:
```suggestion
     "hasMore": boolean
```

We must also describe what this field is in the `Details` section below. You can check out earlier occurrences of it in the document.

Reply:
Additional information was added. :)